packages/react-composites/src/composites/ChatComposite/file-sharing/FileCard.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Icon, mergeStyles, ProgressIndicator, Stack, Text, useTheme } from '@fluentui/react';
+import { getFileTypeIconProps } from '@fluentui/react-file-type-icons';
+import React from 'react';
+
+/**
+ * @beta
+ */
+export interface FileCardProps {
+  /**
+   * File name.
+   */
+  fileName: string;
+  /**
+   * Extension of the file used for rendering the file icon.
+   */
+  fileExtension: string;
+  /**
+   * File upload/download progress percentage between 0 and 1.
+   * Not shown if undefined or 1.
+   */
+  progress?: number;
+  /**
+   * Icon to display for actions like download, upload, etc. on the right of file name.
```

Comment:
I wouldn't indicate where the icon is displayed ("on the right of") as that isn't really specified by the API.

Reply:
Right, but it let's a user know which icon this API changes?

Reply:
Could just say `Icon to display for actions like download, upload et. along with the file name`. There is only one icon in the `FileCard` :)

Reply:
So what's the difference between `along the file name` and `on the right of file name` ? :D

Reply:
It doesn't say where the icon will show up on the UI (for instance, should it be on the left in rtl mode?)

Reply:
Right. That's a good point.